generated_golden.sh

```diff
@@ -52,6 +52,7 @@ scaffold_test_project() {
 		$kb create api --group policy --version v1beta1 --kind HealthCheckPolicy --example=false --controller=true --resource=true --namespaced=false --make=false
 	elif [ $version == "2" ]; then
 		$kb create api --group crew --version v1 --kind FirstMate --controller=true --resource=true --make=false
+		$kb create api --group crew --version v1 --kind SecondMate --controller=true --resource=true --make=false
```

Comment:
I use `crew Captain` in https://github.com/kubernetes-sigs/kubebuilder/pull/644/files#diff-59d90b4ae2a1649c75c68b352dcb3a49R54.

Reply:
We can drop this line now.
You will see `crew Captain` after rebase.

Reply:
Removed the `SecondMate` kind and rebased.